src/napari/components/_tests/test_zoom.py

```diff
@@ -0,0 +1,22 @@
+"""Zoom overlay."""
+
+import pytest
+
+from napari._pydantic_compat import ValidationError
+from napari.components.overlays.zoom import ZoomOverlay
+
+
+def test_zoom():
+    """Test creating zoom object"""
+    zoom = ZoomOverlay()
+    assert zoom is not None
+
+
+def test_zoom_values():
+    """Test creating zoom object"""
+    zoom = ZoomOverlay()
+    # validate canvas_positions
+    zoom.canvas_positions = ((0, 0), (300, 200))
+    with pytest.raises(ValidationError):
+        # data_positions must be 3-D
```

Comment:
```suggestion
        # data_positions must be 2-D
```
? 
As it is expected to fail?

Reply:
Yes - just making sure the validator is working as expected.